CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.9.0](https://github.com/sonata-project/SonataNewsBundle/compare/3.8.0...3.9.0) - 2019-09-02
+### Added
+- Add missing `twig/twig` required in `composer.json` with versions `^1.35 || ^2.4`
+- Added events to the comment process
+- Added a new test class `SonataNewsExtensionTest`. Inspired by the
+  SonataMediaExtensionTest,  `AbstractExtensionTestCase` was extended to test
+if the association with media, user, tag, collection have been created
+according to a configuration.
+- A dependency to [matthiasnoback/symfony-dependency-injection-test
+](https://packagist.org/packages/matthiasnoback/symfony-dependency-injection-test)
+
+### Fixed
+- deprecation notice about using namespaced classes from `\Twig\`
+
+### Changed
+- `src/DependencyInjection/SonataNewsExtension.php` registerDoctrineMapping to
```

Comment:
What?

Reply:
Comes from this PR: https://github.com/sonata-project/SonataNewsBundle/pull/518

Reply:
I think you could just use the PR title:
```diff
-- `src/DependencyInjection/SonataNewsExtension.php` registerDoctrineMapping to
-   add a `trigger_error` message
+ Trigger an error if a class declaration is missing in the configuration
```